tools/vsimporter/xib2nib/UILabel.cpp

```diff
@@ -153,6 +188,10 @@ void UILabel::ConvertStaticMappings(NIBWriter* writer, XIBObject* obj) {
         AddInt(writer, "UINumberOfLines", _numberOfLines);
     }
 
+    if (_lineBreakMode != 4) {
```

Comment:
ok; can you file a bug to terry about this though?

---
In reply to: [96535041](https://github.com/Microsoft/WinObjC/pull/1701#discussion_r96535041) [](ancestors = 96535041,96054766)